parsl/configs/kubernetes.py

```diff
@@ -0,0 +1,41 @@
+from parsl.config import Config
+from parsl.executors import HighThroughputExecutor
+from parsl.providers import KubernetesProvider
+from parsl.addresses import address_by_route
+
+
+config = Config(
+    executors=[
+        HighThroughputExecutor(
+            label='kube-htex',
+            cores_per_worker=1,
+            max_workers=1,
+            worker_logdir_root='YOUR_WORK_DIR',
+
+            # Address for the pod worker to connect back
+            address=address_by_route(),
+            provider=KubernetesProvider(
+                namespace="default",
+
+                # Docker image url to use for deployment
```

Comment:
Better to say `for pods` rather than `for deployment`?

Reply:
Fixed.